core/src/components/button/test/basic/button.e2e.ts

```diff
@@ -10,3 +10,26 @@ test.describe('button: basic', () => {
     expect(await page.screenshot({ fullPage: true })).toMatchSnapshot(`button-diff-${page.getSnapshotSettings()}.png`);
   });
 });
+
+test.describe('button: ripple effect', () => {
+  test('should not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/button/test/basic?ionic:_testing=false&ionic:mode=md`);
+
+    const button = page.locator('#default');
+
+    await button.scrollIntoViewIfNeeded();
+
+    const boundingBox = await button.boundingBox();
+
+    if (boundingBox) {
+      await page.mouse.move(boundingBox.x + boundingBox.width / 2, boundingBox.y + boundingBox.height / 2);
+      await page.mouse.down();
+    }
```

Comment:
Shouldn't playwright do this for you automatically?

I know it scrolls the element into view, but I also thought it clicked in the center of the element by default? https://playwright.dev/docs/input#mouse-click

Reply:
Using `click()` would release the mouse down. This forces the mouse to remain pressed while the screenshot is being captured. It may be indifferent to the captured output. I can try swapping it out to a click, if we think the `waitForSelector` will make it capture the image consistently.

Reply:
Ah, right! Ok should be fine as is then.